app/pages/Help.js

```diff
@@ -19,56 +20,361 @@ class Help extends React.Component {
+            <Icon name="dropdown" />
+            Nox Setup
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 3}>
+            <p>
+              <ol>
+                <a href="https://youtu.be/ZLXJgx-cLNg?t=222" target="_blank">
+                  {' '}
+                  Nox setup video from XennialGaming
+                </a>
+                <li>
+                  Open Nox
+                  <ul>
+                    <li>Open Multi-Instance Manager (icon on the right toolbar of nox)</li>
+                    <li>Create new Android 5 Instance (Name this so you remember)</li>
+                    <li>Close existing instance and open Android 5 Instance</li>
+                    <li>Install Summoners War: Open once for downloading and to sign, in then close Summoners War</li>
+                    <li>Open Settings (top right) and enable root (restart nox)</li>
+                  </ul>
+                </li>
+                <li>
+                  Open SWEX
+                  <ul>
+                    <li>Press "Get Cert" (top right)</li>
+                    <li>Copy file path for certificate (don't include filename)</li>
+                    <li>Navigate to folder with certificate</li>
+                    <li>Drag certificate to Nox homescreen</li>
+                  </ul>
+                </li>
+                <li>
+                  In Nox
+                  <ul>
+                    <li>Tools → Settings → Security → Screen lock: Create a pin</li>
+
+                    <li>Tools → Settings → Security → Insall from SD Card</li>
+
+                    <li>Navigate to: Internal storage → Pictures</li>
+
+                    <li>Click on ca.pem and name it ca.pem</li>
+
+                    <li>Return to home screen</li>
+
+                    <li>Tools → Open file manager and navigate to: /data/misc/keystore/user0 and select certificate (checkbox to the left)</li>
+
+                    <li>With file selected navigate to: /system/etc/security/cacerts</li>
+
+                    <li>Press triple dots in top right (sometimes its bottom left) and select "move selection here"</li>
+
+                    <li>Remove pin (Tools - Tools → Settings → Security → Screen lock)</li>
+
+                    <li>Disable Root (restart Nox)</li>
+                  </ul>
+                </li>
+                <li>
+                  In Nox
+                  <ul>
+                    <li>Tools → Settings → Wi-Fi then long press on network, select "modify network" and then open Advanced options.</li>
+                    <li>Change Proxy to Manual and enter proxy hostname (ip address) and proxy port from SWEX.</li>
+                    <li>Open Summoners War and pray you didn't fuck anything up!</li>
+                  </ul>
+                </li>
+              </ol>
+            </p>
+          </Accordion.Content>
+
+          {/* iOS setup */}
```

Comment:
Not a huge fan of these meaningless comments, since you can see the section directly under it anyway. Would suggest to remove these.